bitriseio/bitrise/profile.go

```diff
@@ -215,12 +215,10 @@ func (client *Client) RegisterProvisioningProfile(provisioningProfSize int64, ex
 }
 
 // UploadProvisioningProfile ...
-func (client *Client) UploadProvisioningProfile(uploadURL string, uploadFileName string, outputDirPath string, exportFileName string) error {
-	log.Printf("Upload %s to Bitrise...", exportFileName)
+func (client *Client) UploadProvisioningProfile(uploadURL string, uploadFileName string, content io.Reader) error {
```

Comment:
Same comment and previously: `uploadFileName` is used only in the logging statement.